src/single_time_series.jl

```diff
@@ -6,12 +6,13 @@
         internal::InfrastructureSystemsInternal
     end
 
-A deterministic forecast for a particular data field in a Component.
+A single column of time series data for a particular data field in a Component.
```

Comment:
We should explain this better 

```suggestion
A single column of time series data for a particular data field in a Component. This is used to represent a single realization or scenario for the time series. 
```

Reply:
How about: `In contrast with a forecast, this can represent one continual time series,
such as a series of historical measurements or realizations or a single scenario
(e.g. a weather year or different input assumptions).`

Reply:
sounds good

Reply:
Done